### DISCUSSION TITLE
Lyric Water Leak Detectors

### DISCUSSION THREAD

**Discussion by balthazzarr** [d0n4v4nb3ck3r/homebridge-honeywell-home#502] [Ideas]
Hi

Is there any chance of adding support for the Lyric water leak detectors?

The devices appear in Homebridge, but nothing happens if a leak is detected.

Thanks

**Comment by donavanbecker** [comment]
I have a leak detector and works for me. Depending on your refreshRate it will take that long to register the leak.

**Reply by balthazzarr** [reply]
Thanks for the reply. Hmm... that's strange. I set the refresh to the minimum 120 seconds, but still nothing after 5+ minutes. I will have to tinker and try and work out what's wrong.
